getnacelle/nacelle-js
# feat: shopify-cart in nuxt starter

### WHY are these changes introduced?

Fixes [[ENG-7070](https://nacelle.atlassian.net/browse/ENG-7070)] 



### WHAT is this pull request doing?

Adds shopify cart to the nuxt starter
Replace checkout with shopify cart checkout


### How to Test

- `git checkout ENG-7070--nuxt-starter-shopify-cart`
- to the nuxt reference store project `.env` add the following: (reach out if you want the vars I used)

```
SHOPIFY_SHOP_ID="<YOUR_SHOPIFY_ID>"
SHOPIFY_STOREFRONT_ACCESS_TOKEN="<YOUR_SHOPIFY_STOREFRONT_ACCESS_TOKEN>"
```

- `cd starter/nuxt`
- `npm i && npm run generate && npm run start`
- see if cart/checkout works
